_includes/sections/browser-recommendation.html

```diff
@@ -22,14 +22,109 @@ <h1 id="browser" class="anchor"><a href="#browser"><i class="fas fa-link anchor-
+description="Tor Browser is your choice if you need an extra layer of anonymity. It\'s a modified version of Firefox ESR, which comes with pre-installed privacy add-ons, encryption and an advanced proxy. <a href='https://write.privacytools.io/my-thoughts-on-security/slicing-onions-part-1-myth-busting-tor'>How does Tor work?</a>"
 website="https://www.torproject.org/"
 forum="https://forum.privacytools.io/t/discussion-tor-browser/278"
 git="https://trac.torproject.org/projects/tor"
 windows="https://www.torproject.org/download/"
 mac="https://www.torproject.org/download/"
 linux="https://www.torproject.org/download/"
+%}
+
+<h1 id="browser-android" class="anchor">
+  <a href="#browser-android"><span class="fas fa-link anchor-icon"></span></a>
+  Browser Recommendations For Android
+</h1>
+
+{% include cardv2.html
+title="Firefox"
+image="/assets/img/tools/Firefox.png"
+description="Firefox is fast, reliable, open source and respects your privacy. Don\'t forget to adjust the settings according to our
+recommendations: <a href='#webrtc'><i class='fas fa-link'></i> WebRTC</a> and <a href='#about_config'><i class='fas fa-link'></i> about:config</a> and get the <a href='#addons'><i class='fas fa-link'></i> privacy add-ons</a>."
+website="https://www.mozilla.org/en-US/firefox/mobile/"
+forum="https://forum.privacytools.io/t/discussion-firefox/279"
+source="https://github.com/mozilla-mobile"
+fdroid="https://f-droid.org/en/packages/org.mozilla.fennec_fdroid/"
+googleplay="https://play.google.com/store/apps/details?id=org.mozilla.firefox"
+%}
+
+{% include cardv2.html
+title="Tor Browser - Provides Anonymity"
+image="/assets/img/tools/Tor-Browser.png"
+description="Tor Browser is your choice if you need an extra layer of anonymity. It\'s a modified version of Firefox ESR, which comes with pre-installed privacy add-ons, encryption and an advanced proxy. <a href='https://write.privacytools.io/my-thoughts-on-security/slicing-onions-part-1-myth-busting-tor'>How does Tor work?</a>"
+Website="https://torproject.org"
+forum="https://forum.privacytools.io/t/discussion-tor-browser-for-android-browsers/1522"
+git="https://gitweb.torproject.org/tor-browser.git/"
 android="https://www.torproject.org/download/#android"
 fdroid="https://guardianproject.info/fdroid/"
 googleplay="https://play.google.com/store/apps/details?id=org.torproject.torbrowser"
 %}
+
+{% include cardv2.html
+title="Bromite"
+image="/assets/img/tools/bromite.png"
+description="Bromite is a Chromium-based browser with security enhancedment patches from GrapheneOS and built-in adblocking and DNS over HTTPS support. More info can be found <a href='https://www.bromite.org/#main-features'>here</a>."
+website="https://www.bromite.org/"
+forum="https://forum.privacytools.io/t/discussion-bromite-browsers/1521"
+github="https://github.com/bromite/bromite"
+android="https://www.bromite.org/news/2018-12-21-new-fdroid-repository-url"
```

Comment:
```suggestion
fdroid="https://www.bromite.org/fdroid"
android="https://www.bromite.org/#download-bromite"
```

Android is for APK download